docs/src/configs/docs/players/entities.js

```diff
@@ -6,38 +6,43 @@ export default [
       {
         field: 'id',
         type: 'integer',
-        description: "The player's unique id.",
+        description: "The player's unique id."
       },
       {
         field: 'username',
         type: 'string',
-        description: "The player's unique username.",
+        description: "The player's unique username."
       },
       {
         field: 'type',
         type: 'string',
-        description: "The player's account type. (See possible values below)",
+        description: "The player's account type. (See possible values below)"
       },
       {
         field: 'lastImportedAt',
         type: 'date',
-        description: "The last time this player's history was imported from CML.",
+        description: "The last time this player's history was imported from CML."
       },
       {
         field: 'registeredAt',
         type: 'date',
-        description: "The player's registration date.",
+        description: "The player's registration date."
       },
       {
         field: 'updatedAt',
         type: 'date',
-        description: "The player's last update date.",
+        description: "The player's last update date."
       },
-    ],
+      {
+        field: 'combatLevel',
```

Comment:
Hum the combatLevel isn't really a column in the players table, it's just a calculated field in the player details endpoint

Reply:
I must have been quick and thought it was the response fields of the `.../api/players/?id={x}` endpoint. Will remove this part.